src/overlays/actors/ovl_Obj_Tokei_Step/z_obj_tokei_step.c

```diff
@@ -145,25 +141,25 @@ s32 ObjTokeiStep_OpenProcess(ObjTokeiStep* this, GlobalContext* globalCtx) {
     s32 i;
     ObjTokeiStepPanel* panel;
     f32 finalPosY;
-    s32 isOpen = 1;
-    s32 prevBounced = 1;
+    s32 isOpen = true;
+    s32 didPrevBounce = true;
```

Comment:
Could you explain what `didPrevBounce` does? Everything else in this PR seems good. This name seems just a little awkward to me though

Reply:
Its a boolean whether the step in front (previous in the array) of the current one had bounced up at least once. Basically in the cutscene of the clockertower opening, as each step opens up it will drop below its final y position, then bounce back up slightly a total of 3 times before stopping. On the first of these bounces, a timer of the next step starts counting down from 10. When that reaches 0 it starts to open.

Reply:
Hmm I think I get it, thanks. What about something like `hasBounced`? Or `hasPrevBounced`? You don't have to take those if you don't want, it'll be up to you

Reply:
I do like `hasPrevBounced` better.